Source/Scene/Vector3DTileContent.js

```diff
@@ -0,0 +1,324 @@
+            this.state = Cesium3DTileContentState.READY;
+            this.readyPromise.resolve(this);
+            return;
+        }
+
+        var featureTableJSONByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+        var featureTableBinaryByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+        var batchTableJSONByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+        var batchTableBinaryByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+        var indicesByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+        var positionByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+
+        var featureTableString = getStringFromTypedArray(uint8Array, byteOffset, featureTableJSONByteLength);
+        var featureTableJSON = JSON.parse(featureTableString);
+        byteOffset += featureTableJSONByteLength;
+
+        var featureTableBinary = new Uint8Array(arrayBuffer, byteOffset, featureTableBinaryByteLength);
+        byteOffset += featureTableBinaryByteLength;
+
+        var numberOfPolygons = featureTableJSON.NUMBER_OF_POLYGONS;
+
+        var batchTableResources = new Cesium3DTileBatchTableResources(this, numberOfPolygons, createColorChangedCallback(this));
+        this.batchTableResources = batchTableResources;
+        if (batchTableJSONByteLength > 0) {
+            var batchTableString = getStringFromTypedArray(uint8Array, byteOffset, batchTableJSONByteLength);
+            batchTableResources.batchTable = JSON.parse(batchTableString);
+            byteOffset += batchTableJSONByteLength;
+        }
+
+        // TODO: Right now batchTableResources doesn't support binary
+        byteOffset += batchTableBinaryByteLength;
+
+        var indices = new Uint32Array(arrayBuffer, byteOffset, indicesByteLength / sizeOfUint32);
+        byteOffset += indicesByteLength;
+        var positions = new Uint16Array(arrayBuffer, byteOffset, positionByteLength / sizeOfUint16);
+
+        byteOffset = featureTableBinary.byteOffset + featureTableJSON.POLYGON_POSITION_OFFSETS.offset;
+        var offsets = new Uint32Array(featureTableBinary.buffer, byteOffset, numberOfPolygons);
+
+        byteOffset = featureTableBinary.byteOffset + featureTableJSON.POLYGON_POSITION_COUNTS.offset;
+        var counts = new Uint32Array(featureTableBinary.buffer, byteOffset, numberOfPolygons);
+
+        byteOffset = featureTableBinary.byteOffset + featureTableJSON.POLYGON_INDICES_OFFSETS.offset;
+        var indexOffsets = new Uint32Array(featureTableBinary.buffer, byteOffset, numberOfPolygons);
+
+        byteOffset = featureTableBinary.byteOffset + featureTableJSON.POLYGON_INDICES_COUNTS.offset;
+        var indexCounts = new Uint32Array(featureTableBinary.buffer, byteOffset, numberOfPolygons);
+
+        var center = Cartesian3.unpack(featureTableJSON.CENTER);
```

Comment:
In https://github.com/AnalyticalGraphicsInc/cesium/pull/4183 I call this `RTC_CENTER`. Also perhaps this property should be optional...